integrations/artifacts/fluxhelmresource-crd.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: apiextensions.k8s.io/v1beta1
```

Comment:
Oh right, this is the canonical version of this file. It'd be good to have it only in one place. See my comments in deploy/helm-operator.yaml above.

It'd be good to have this only in one place. Is there a downside to the operator applying it on startup?

Reply:
yes, apiextensions.k8s.io/v1beta1 is the k8s API version for creating CRDs. I shall make the changes you suggested for deploy/helm-operator.yaml.

I don't think there is a downside to that..

Reply:
The comment's there because that's the first line of the file :)

Reply:
Actually there is a preferred sequence as to how the three parts of the integration functionality are released:
1) CRD ... both the fluxd and helm-operator depend on it
2) fluxd ... help-operator depends on it for the secret
3) helm-operator

Saying that,  helm-operator can apply the CRD at the startup and wait for fluxd to create the keys.